whylog_vim/gui/files_manager.py

```diff
@@ -0,0 +1,28 @@
+import vim
+from whylog_vim.gui.vim_commander import VimCommander
+
+
+class FilesManager(object):
+
+    @staticmethod
+    def get_files_window_id(file_name):
+        for window_id, window in enumerate(vim.windows):
+            if window.buffer.name.endswith(file_name):
+                return window_id + 1
```

Comment:
Are you sure that that ids giving is a good mapping between number and file? What if there will be removal of window in vim.windows? I suppose ids will be then different than before removal.